etc/config/javascript.defaults.properties

```diff
@@ -0,0 +1,19 @@
+# Default settings for Javascript
+
+compilers=javascript
+defaultCompiler=javascript
+instructionSet=d8
+
+supportsBinary=false
+interpreted=true
+compilerType=d8
+objdumper=
+demangler=
+postProcess=
+options=
+supportsExecute=false
+stubText=function add(a, b) { return a + b; } for (var i = 0; i < 10; i++) { add(i, 1) }
+
+compiler.javascript.exe=/home/gaurav/v8/out.gn/x64.debug/d8
```

Comment:
I know this can't be.... but I don't know where to put this. Please tell me.

Reply:
We've not really got a good answer: maybe just `d8` and let the `$PATH` find it by default.